contents/data_vis.md

```diff
@@ -368,9 +368,157 @@ What more can we ask for?
+However, sometimes is better to specified the color that you want at both ends. We do that with `highclip` and `lowclip`:
+
+```
+using ColorSchemes
+```
+
+```jl
+s = """
+    figure = (;resolution = (400,300), font= "CMU Serif")
+    axis = (; xlabel = L"x", ylabel = L"y", aspect= DataAspect())
+
+    fig, ax, pltobj = CairoMakie.heatmap(randn(20,20); colorrange = (-2,2),
+        colormap = "diverging_rainbow_bgymr_45_85_c67_n256",
+        highclip = :black, lowclip = :white, axis = axis, figure = figure)
+    Colorbar(fig[1,2], pltobj, label = "Diverging colormap")
+    fig
+    label = "diverging_colormap" # hide
+    caption = "Diverging Colormap with low and high clip." # hide
+    Options(fig; filename=label, label, caption) # hide
+    """
+sco(s)
+```
+
+But we mentioned that also RGB vectors are valid options.
+For our next example you could pass the custom colormap _perse_ or use `cgrad` to force a categorical colorbar.
+
+```
+using ColorSchemes
+```
+
+```jl
+scat = """
+    figure = (;resolution = (400,300), font= "CMU Serif")
+    axis = (; xlabel = L"x", ylabel = L"y", aspect= DataAspect())
+    cmap = ColorScheme(range(colorant"red", colorant"green", length=3))
+    mygrays = ColorScheme([RGB{Float64}(i, i, i) for i in [0.0,0.5,1.0]])
+
+    fig, ax, pltobj = CairoMakie.heatmap(rand(-1:1,20,20);
+        colormap = cgrad(mygrays, 3, categorical = true, rev = true), # cgrad and Symbol, mygrays,
+        axis = axis, figure = figure)
+    cbar = Colorbar(fig[1,2], pltobj, label = "Categories")
+    cbar.ticks = ([-0.66,0,0.66], ["-1","0","1"])
+    fig
+    label = "categorical_colormap" # hide
+    caption = "Categorical Colormap." # hide
+    Options(fig; filename=label, label, caption) # hide
+    """
+sco(scat)
+```
+
+Lastly, the ticks in the colorbar for the categorial case are not centered by default in each color.
+This is fixed by passing custom ticks, as in `cbar.ticks = (positions, ticks)`.
+The last situation is when passing a `tuple` of two colors to `colormap` as symbols, strings or a mix.
+You will get an interpolated colormap between these two colors.
+
+Also, `hex` coded colors are also accepted. So, on top or our heatmap let's put one semi-transparent point using this.
+
+```jl
+s2color2 = """
+    figure = (;resolution = (400,300), font= "CMU Serif")
+    axis = (; xlabel = L"x", ylabel = L"y", aspect= DataAspect())
+
+    fig, ax, pltobj = CairoMakie.heatmap(rand(20,20); colorrange = (0,1),
+        colormap = (:red, "black"), axis = axis, figure = figure)
+    CairoMakie.scatter!(ax,[11],[11],color=("#C0C0C0", 0.5),markersize=150)
+    Colorbar(fig[1,2], pltobj, label = "2 colors")
+    fig
+    label = "colormap_two_colors" # hide
+    caption = "Colormap from two colors." # hide
+    Options(fig; filename=label, label, caption) # hide
+    """
+sco(s2color2)
+```
+
+
+ **Custom cycle**
+
+Here, we could define a global `Theme` with a new cycle for colors, however that is not the recommend way to do it.
```

Comment:
```suggestion
Here, we could define a global `Theme` with a new cycle for colors, however that is **not the recommend way** to do it.
```

Always have to be careful with these not stuff in the middle of the sentence. I have high-school traumas from nasty questions.